framerail/src/routes/[x+2d]/user/[slug]/page.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { page } from "$app/stores"
+</script>
+
+<h1>UNTRANSLATED: Loaded user profile</h1>
+
+<textarea class="debug">{JSON.stringify($page, null, 2)}</textarea>
+
+<div class="user-info" data-id={$page.data.user.user_id}>
+  {#if $page.data.user.name}
+    <h2 class="user-attribute name">
+      {$page.data.user.name}
+    </h2>
+  {/if}
+
+  {#if $page.data.user.real_name}
+    <div class="user-attribute real-name">
+      {$page.data.internationalization?.["user-profile-info.name"]}
+      {$page.data.user.real_name}
```

Comment:
General note/nitpick, I think it would be helpful for CSS themers to wrap loose text in spans